.github/workflows/php-tests-wp-4-7-php-5-6.yml

```diff
@@ -0,0 +1,69 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  php-tests-wp-4-7-php-5-6:
+    name: WP 4.7, PHP 5.6
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
```

Comment:
@tofumatt MySQL service isn't needed because there is a MySQL server on the virtual machine itself. Just run sudo systemctl start mysql.service to start it and you can use it with root user and root password.

Reply:
@aaemnnosttv I don't have such doubts and think that it should work without any problem. Anyway, we can try it with the available version of MySQL server on the host and if it doesn't work for WordPress 4.7, then switch back to using a service, ok?

Reply:
@eugene-manuilov My concern there is that while it may work today, unexpected changes to the MySQL version on the runner could break things in the future. The [requirements page](https://make.wordpress.org/hosting/handbook/handbook/server-environment/#database) shows it supports MySQL `8.0.x` – does that mean older versions might not work with MySQL 8.1?

There are already a few tickets around MySQL 8:
- https://core.trac.wordpress.org/ticket/51740
- https://core.trac.wordpress.org/ticket/49364

It seems like it would be a good idea to stick to the latest MySQL v5 for now.

Reply:
I wish we could try MySQL 8 for our tests anyway and use it if it doesn't cause problems.

Reply:
The existing ones, I thought, assumed v5, so let's stick with that for now but add a v8 test (or move to it) in another issue? I'm just wary of introducing changes to the tests here during the migration.